lib/utils.js

```diff
@@ -769,7 +775,7 @@ function relayEvents(listener, emitter, events) {
  * instances
  *
  * @private
- * @param {(Topology|Server)} topologyOrServer
+ * @param {Topology | Server | any} topologyOrServer TODO:(ts) better types
```

Comment:
where is `any` passed in? I think that's probably a bug.

Reply:
Topology doesn't have the property `ismaster`, Server doesn't have the type `lastIsMaster()` so I have to put type assertions on each access or just here at the top of the function, by "better type" and I should be more explicit we should come up with a better way to assert whether its a Server or Topology.